sample_tracking/app/all-users/page.tsx

```diff
@@ -184,46 +186,89 @@ export default function Users() {
     );
 
     function handleRemoveClick(userData: UserData) {
-        const removedMemberId = userData.user_id;
-        const confirmString = `Are you sure you want to remove ${userData.name}?`
-        if (!confirm(confirmString)) return;
-        deleteDoc(doc(db, "users", removedMemberId));
-        setDoc(doc(db, "new_users", removedMemberId), {
-            email: userData.email,
-            uid: userData.user_id,
-            name: userData.name
-        })
-        if (userData.email) {
-            updateDoc(doc(db, "organizations", userData.org), {
-                members: arrayRemove(userData.email)
+
+        const removeUserFunction = () => {
+            const removedMemberId = userData.user_id;
+            deleteDoc(doc(db, "users", removedMemberId));
```

Comment:
you need to catcherror here and display any errors that may have happened attempting the operation.
you can test this by turning off internet just as you hit delete and see what happens